erpnext/controllers/stock_controller.py

```diff
@@ -329,6 +329,7 @@ def validate_warehouse(self):
 		warehouses.extend(from_warehouse)
 
 		for w in warehouses:
+			is_disabled_warehouse(w)
```

Comment:
```suggestion
			validate_disabled_warehouse(w)
```
`is_disabled_warehouse` kind of implies that it will return a true/false value back like `is_perpetual_inventory_enabled`.
If its a function that **primarily** carries out a task within itself then it's best to describe what that function is doing.